android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UriBeaconDiscoveryService.java

```diff
@@ -475,14 +475,15 @@ private RemoteViews updateSummaryNotificationRemoteViews() {
   }
 
   private void updateSummaryNotificationRemoteViewsFirstBeacon(String url, RemoteViews remoteViews) {
-    MetadataResolver.UrlMetadata urlMetadata_firstBeacon = mUrlToUrlMetadata.get(url);
-    if (urlMetadata_firstBeacon != null) {
-      String title = mUrlToUrlMetadata.get(url).title;
-      String description = mUrlToUrlMetadata.get(url).description;
-      Bitmap icon = mUrlToUrlMetadata.get(url).icon;
+    MetadataResolver.UrlMetadata urlMetadata = mUrlToUrlMetadata.get(url);
+    if (urlMetadata != null) {
+      String title = urlMetadata.title;
```

Comment:
I don't think there is a need for these variables. `urlMetadata.*` seems short and clear enough and they are only used once. Sorry to make you do all this clean up work :)